site/_guides/health-checking.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Health Checking
+layout: page
+---
+
+<div id="toc" class="navigation"></div>
```

Comment:
Did you use the `make site-devel` target to preview the docs? You should not need a TOC here, since there are no header elements in the page.